ansible_collections/arista/avd/examples/campus-fabric/inventory.yml

```diff
@@ -0,0 +1,37 @@
+# inventory.yml
+DC1:
+  children:
+    DC1_FABRIC:
+      children:
+        DC1_SPINES:
+          hosts:
+            SPINE1:
+              ansible_host: 172.100.100.101
```

Comment:
Do we need the `ansible_host` here?

Reply:
I use it to push configs to a virtualized environment...

Reply:
should it not be included as part of this example?

Reply:
They are used in the previous L2LS example.

https://avd.sh/en/devel/examples/l2ls-fabric/index.html